Rails-Girls-Summer-Of-Code-2017-Blog-Post-2.md

```diff
@@ -0,0 +1,53 @@
+
+On our third meeting with Kelsey, she had said that "It is better to over-communicate rather than Under-communicate" and thus, we would ( shamelessly 🙈 ) ping them on slack or on the issues whenever we were stuck and a descriptive solution to our questions would follow. Many a times, Kelsey would explain us our doubts on our call. Despite whatever mistakes we would make, the committee members were always positive and ensured that we were able to rectify our errors and learn from them. Many a times, they've tagged us in issues that they thought we should be aware of and sent us links and resources to escalate our learning.
+
+Tessel has provided us amazing Mentorship, one that is really hard to find. We are so glad that we began our journey with Open Source with such a welcoming community, so much so that we are more than willing to work for Tessel even after RGSoC is over. Tessel, We love you!
+
+## Extras
+
+In these three months, we not only focussed on core development work but we also went on presenting a few lightning talks. We gave two of them - one for Women Who Code, Delhi and another for LinuxChix India. The LinuxChix India Meetup for focussed more on Open Source Contributions, RGSoC and Tessel. Our supervisor Vaishali had introduced us to the community and put us in touch with the organizer of the meetup. We spoke about RGSoC - How to apply, what all do we do, our work, our social media, our blog posts, in and all everything. After that, we went on to demonstrate our project - Tessel. There were several Open Source enthusiasts who had loads of questions about Tessel and we were more than delighted to answer them. One person even wanted to use Tessel in his project that he was thinking of starting!
+
+It was a great experience for both of us because we were able to pass on our knowledge to an enthusiastic crowd, get rid of our fear of speaking in front of unknown people and gaining in depth knowledge of the topics involoved for which we had to explore every nook and corner of them in order to make our sessions productive.
+
+## What's next?
+
+RGSoC provided us with a whole lot of things that we not even imagined of gaining in this interval of three months. As it was a wonderful journey for both of us, we have thought of recommending this program to more and more people by conducting a few related meetups at our University under our Rails Girls Summer of Code, Delhi Chapter. Apart from this, we had discussed way back with our mentor during the application process that we would love to contribute to Tessel even after RGSoC ends. The months might be over, but the memories are here to stay!
+
+Who would know that two confused, scared and shy humans who had absolutely no idea how they would survive the tech world could successfully complete RGSoC, loaded with experience and confidence?
```

Comment:
I'm so glad! Thanks for giving me the opportunity to work with you, I really got a lot out of it too.

Reply:
Thanks a lot for the reviews. Would be updating the changes soon.